### PR TITLE
add secret scanning exclusion tuning

### PR DESCRIPTION
**Pull request by SadFaceSmith**
**What is this feature?**

Adds a tuning file to ignore test/example paths during Secret Scanning

**Why do we need this feature?**

To lessen the amount of false positive alerts to repo owners. 

**Who is this feature for?**

Repo owners

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by tolzhabayev** [comment]
Please rebase on `main` to make this go away:
SCR-20230320-ked

**Comment by SadFaceSmith** [comment]
> Please rebase on `main` to make this go away: SCR-20230320-ked

Done 👍

**Review by tolzhabayev — APPROVED** [review]

**Comment by SadFaceSmith** [comment]
@tolzhabayev Is there a way to re-run the CODEOWNER validator check?

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -594,6 +594,8 @@ embed.go @grafana/grafana-as-code
 /.github/workflows/publish-technical-documentation-release.yml @grafana/docs-grafana
 /.github/workflows/remove-milestone.yml @grafana/user-essentials
 /.github/workflows/sbom-report.yml @grafana/security-team
+/.github/secret_scanning.yml @grafana/security-team
+/.github/workflows/snyk.yml @grafana/security-team
```
Comment (tolzhabayev) [inline .github/CODEOWNERS:598]: this is duplicate of https://github.com/grafana/grafana/blob/360c652e39d33c27214a5ccec0176164bc17e740/.github/CODEOWNERS#L603

Comment (SadFaceSmith) [reply .github/CODEOWNERS:598]: Ah of course. Thanks! I removed the duplicate on line 603

**Pull request closed without merging**